### PR TITLE
fix: clipbardItem不存在时不报错

### PR DESCRIPTION
**Pull request by serializedowen**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by YardWill — APPROVED** [review]
LG

File: packages/s2-core/src/utils/export/index.ts [hunk 1]
```diff
@@ -61,6 +61,11 @@ export const copyToClipboardByExecCommand = (data: Copyable): Promise<void> => {
 };
 
 export const copyToClipboardByClipboard = (data: Copyable): Promise<void> => {
+  // https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem
+  if (!ClipboardItem) {
```
Comment (wjgogogo) [inline packages/s2-core/src/utils/export/index.ts:65]: 加在这里要合适点：
https://github.com/antvis/S2/blob/cfcdacf94717f4a590b0f1e4424f51180f055ce6/packages/s2-core/src/utils/export/index.ts#L96

Comment (serializedowen) [reply packages/s2-core/src/utils/export/index.ts:65]: ok

**Pull request merged**